### PR TITLE
Update adapter to follow prebid conventions

### PR DESCRIPTION
**Pull request by samueldobbie**
Related issue: https://github.com/glimpse-protocol/meta/issues/44

### CODE REVIEW

File: modules/glimpseBidAdapter.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const storageManager = getStorageManager()
 
 const GVLID = 1012
 const BIDDER_CODE = 'glimpse'
-const ENDPOINT = 'https://api.glimpsevault.io/ads/serving/public/v1/prebid'
+const ENDPOINT = 'https://api.glimpsevault.local/ads/serving/public/v1/prebid'
```
Comment (tim-hm) [inline modules/glimpseBidAdapter.js:11]: Did you mean to commit with this pointing at local?

**Review by tim-hm — CHANGES_REQUESTED** [review]
Can you squash the fixup commit and I'd like to ask a few questions about the changes as I'm struggling to keep up with ya!

Otherwise code looks good.l

**Comment by tim-hm** [comment]
One other comment .. for the commit message / body, you could add more about the why of these changes (and provide a link to the git issue discussion on the prebid repo). Just helpful to have that context when reviewing (unless i'm lost here!)

**Review by tim-hm — APPROVED** [review]
Did you mean to modify and commit `package-lock.json`?

**Comment by samueldobbie** [comment]
@tim-hm Elaborated on the changes within the commit message & removed the empty line causing the `package-lock.json` change.

**Pull request merged**